lib/experimental/block-comments.php

```diff
@@ -93,3 +93,277 @@ function exclude_block_comments_from_admin( $clauses, $query ) {
 	}
 	add_action( 'comments_clauses', 'exclude_block_comments_from_admin', 10, 2 );
 }
+
+/**
+ * Post List filters
+ */
+
+/**
+ * Add a new column for editorial comments.
+ *
+ * @param array $columns Existing columns.
+ * @return array Modified columns.
+ */
+function gutenberg_add_editorial_comments_column( $columns ) {
+	// Add column immediately after the existing comments column.
+	$editorial_comments = '<div class="dashicons dashicons-admin-comments" title="' . esc_attr__( 'Editorial Comments', 'gutenberg' ) . '"></div>';
```

Comment:
```suggestion
	$editorial_comments = sprintf(
		'<span class="vers comment-grey-bubble" title="%1$s" aria-hidden="true"></span><span class="screen-reader-text">%2$s</span>',
		esc_attr__( 'Comments', 'gutenberg' ),
		/* translators: Hidden accessibility text. */
		__( 'Comments', 'gutenberg' )
	);
```

How about using the same HTML as the original comment?

https://github.com/WordPress/wordpress-develop/blob/70d00508e950f0e63de82386d453bca5ad899474/src/wp-admin/includes/class-wp-posts-list-table.php#L708-L713